### DISCUSSION TITLE
Why don't people like the "For You" section?

### DISCUSSION THREAD

**Discussion by Scripped** [community/community#13557] [Other Feature Feedback, Questions, & Ideas]
I don't mind it, I think it's good to find things you're interested in.

**Comment by Trebor-Huang** [comment]
Well, people have given their reasons in their corresponding threads. Why this question?

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
